README.md

```diff
@@ -226,6 +226,17 @@ bugsnag.Notify(err,
     bugsnag.User{Id: "1234", Name: "Conrad", Email: "me@cirw.in"})
 ```
 
+### Error Class
+
+Errors in your Bugsnag dashboard are grouped by their "error class".
+You can override the error class by passing a
+[`bugsnag.ErrorClass`](https://godoc.org/github.com/bugsnag/bugsnag-go/#ErrorClass) object as
+rawData.
+
+```go
+bugsnag.Notify(err, bugsnag.ErrorClass{Name: "I/O Timeout"})
```

Comment:
You shouldn't need `Name` in here, you should be able to just have `bugsnag.ErrorClass{"I/O Timeout"}` as the example.